test/unit_tests/25_host_multirate/testSingleOutBuffer.cpp

```diff
@@ -29,58 +29,64 @@ int main(int argc, char *argv[]) {
   mlir_aie_init_device(_xaie);
   mlir_aie_configure_cores(_xaie);
   mlir_aie_configure_switchboxes(_xaie);
+
+  mlir_aie_clear_tile_memory(_xaie, 3, 4);
+
   mlir_aie_configure_dmas(_xaie);
   mlir_aie_initialize_locks(_xaie);
 
-  mlir_aie_init_mems(_xaie, 2);
+  mlir_aie_init_mems(_xaie, 3);
```

Comment:
As we only have one buffer in and one buffer out we should init 2 memories, i.e., `mlir_aie_init_mems(_xaie, 2);`

Reply:
Thanks, fixed.